### PR TITLE
Ref(T32651): Vue migration: adjust AdministrationPlaces component to new version of DpDataTable

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T32651

**Description:** After refactoring DpDataTable, we had to make adjustments to its usage in various places. These changes are necessary to ensure compatibility with the new version of DpDataTable.

### Linked PRs
**Important:** This PR depends on [demosplan-ui#230](https://github.com/demos-europe/demosplan-ui/pull/230) and [demosplan-ui#233](https://github.com/demos-europe/demosplan-ui/pull/233).

It can only be merged after both PRs are merged.

### CODE REVIEW

File: client/js/components/procedure/admin/AdministrationPlaces.vue [hunk 1]
```diff
@@ -309,6 +315,19 @@ export default {
         })
     },
 
+    setEditMode (rowData, state = true) {
+      const idx = this.places.findIndex(el => el.id === rowData.id)
+
+      this.places[idx].edit = state
+    },
+
+    updatePlaceData (rowData) {
```
Comment (salisdemos) [inline client/js/components/procedure/admin/AdministrationPlaces.vue:324]: same here. id instead of rowData

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/admin/AdministrationPlaces.vue (hunk 1 above)
Comment (sakutademos) [reply client/js/components/procedure/admin/AdministrationPlaces.vue]: done here https://github.com/demos-europe/demosplan-core/pull/1487/commits/dce2bdabf97f182b0a38094e0af4228f22d8aec1

Comment (sakutademos) [reply client/js/components/procedure/admin/AdministrationPlaces.vue]: done here https://github.com/demos-europe/demosplan-core/pull/1487/commits/dce2bdabf97f182b0a38094e0af4228f22d8aec1

File: client/js/components/procedure/admin/AdministrationPlaces.vue [hunk 2]
```diff
@@ -309,6 +315,19 @@ export default {
         })
     },
 
+    setEditMode (rowData, state = true) {
```
[the remark this answers is not in the document]
Comment (sakutademos) [reply client/js/components/procedure/admin/AdministrationPlaces.vue]: done here https://github.com/demos-europe/demosplan-core/pull/1487/commits/dce2bdabf97f182b0a38094e0af4228f22d8aec1

**Review by salisdemos — APPROVED** [review]
legit, 

Pay attentionen, that this has to be merged the momement we publish the fitting version of demosplan-ui

**Pull request merged**